peribolos/knative-sandbox.yaml

```diff
@@ -220,6 +220,14 @@ orgs:
     repos:
       # NOTE: If you are adding a repo here, you may also want to grant repo
       # permissions for one or more teams, in the next section.
+      .github:
+        allow_merge_commit: false
+        allow_rebase_merge: false
+        # See also: https://github.blog/changelog/2019-02-21-organization-wide-community-health-files/
+        # See also: https://docs.github.com/en/actions/configuring-and-managing-workflows/sharing-workflow-templates-within-your-organization
+        description: "Repository for sharing org-wide Github metadata and workflows."
```

Comment:
workflow templates. It seems you don't auto-apply all the workflows from this repo unless I misunderstand it

Reply:
maybe mention issue templates too?

Reply:
done

Reply:
that's metadata :)